react-native-twilio-video-webrtc.podspec

```diff
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
   s.source_files   = 'ios/*.{h,m}'
 
   s.dependency 'React'
-  s.dependency 'TwilioVideo', '>= 1.0.1'
+  s.dependency 'TwilioVideo', '>= 1.3.0'
```

Comment:
I don't think we need to change the constraint. If you want to run `1.3.0` you can add the constraint in your application's Podfile